apps/email/index.html

```diff
@@ -32,6 +32,7 @@
   <link rel="stylesheet" type="text/css" href="shared/style/progress_activity.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/tabs.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/value_selector.css" >
+  <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" >
```

Comment:
This is needed for the 'back' buttons that needs mirroring icons in RTL.

This can be costly for just one icon, but I figured that maybe we want to replace a lot of them by gaia-icons. I'm thinking about the attach, send, trash, back, forward icons... basically most icons in style/images/icons have a corresponding one in gaia-icons. And we could add the one that are not yet there.

I see several advantages of doing so:
- gaia-icons.ttf is cached at the system level to speed up the startup of the application (that's what Wilson Page told me at least). We would benefit from the speed gain if we use them.
- It guarantees we stay visually consistent with the rest of gaia.

WDYT? I can open a bug for that if you want.